static/app/components/replays/replayView.tsx

```diff
@@ -50,4 +78,23 @@ const ContextContainer = styled('div')`
   gap: ${space(1)};
 `;
 
+const PlayerContainer = styled('div')`
+  display: grid;
+  grid-auto-flow: row;
+  grid-template-rows: auto 1fr;
+  gap: ${space(1)};
+  width: 100%;
+`;
+
+const BreadcrumbContainer = styled('div')`
+  padding-left: ${space(1)};
+  width: 25%;
+`;
+
+const PlayerBreadcrumbContainer = styled('div')`
+  display: flex;
+  flex-direction: row;
+  height: 100%;
```

Comment:
I'm not sure if the height is needed here.... but besides that, it's flex, so can we give this a `gap: {$space(1)};`, i think if we do that then we can remove the padding-left from BreadcrumbContainer

Reply:
height is needed for the video to be vertically centred and background to show up :(